preview/app/Global.scala

```diff
@@ -0,0 +1,3 @@
+import controllers.PreviewHealthCheckLifeCycle
+
+object Global extends StandaloneGlobal with PreviewHealthCheckLifeCycle
```

Comment:
Necessary to give preview and training-preview a different HealthCheckLifeCycle trait